src/box2dx/Box2D.NetStandard/Collision/BroadPhase.cs

```diff
@@ -54,197 +54,197 @@
 
 namespace Box2D.NetStandard.Collision
 {
-    internal class BroadPhase
```

Comment:
I guess this is all red because I dared to reformat with JetBrains tooling and it's replaced spaces with tabs